src/main/java/com/jvm_bloggers/kafka/message/NewIssuePublishedMessage.java

```diff
@@ -0,0 +1,14 @@
+package com.jvm_bloggers.kafka.message;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.Value;
+
+@KafkaMessage
+@Value
+public class NewIssuePublishedMessage {
+    @JsonProperty("issueNumber")
```

Comment:
They are meant to protect against refactoring (if someone changes field name).

However tests would also detect this too, so I agree - they are redundant.